api/envoy/extensions/quic/udp_packet_writer/v3/udp_default_writer_factory.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package envoy.extensions.quic.udp_packet_writer.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.quic.udp_packet_writer.v3";
+option java_outer_classname = "UdpDefaultWriterFactoryProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/quic/udp_packet_writer/v3;udp_packet_writerv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: UDP packet writer config]
+// [#extension: envoy.quic.udp_packet_writer.server.quiche]
+
+// Configuration for the default UDP packet writer factory.
```

Comment:
Maybe add more details here about the default writer which just uses kernel sendmsg.

Reply:
Good idea. Done.